GetService/handler.ts

```diff
@@ -49,12 +49,12 @@ export function GetServiceHandler(
 ): IGetServiceHandler {
   // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
   return async (_, __, serviceId) => {
-    const errorOrMaybeRetrievedService = await serviceModel
-      .findOneByServiceId(serviceId)
-      .run();
+    const errorOrMaybeRetrievedService = await serviceModel.findOneByServiceId(
```

Comment:
Why not making it functional here?
```
return async (_, __, serviceId) =>
    await pipe(
      serviceModel.findOneByServiceId(serviceId),
      TE.mapLeft(e =>
        ResponseErrorQuery("Error while retrieving the service", e)
      ),
      TE.chainW(o =>
        pipe(
          o,
          TE.fromOption(() =>
            ResponseErrorNotFound(
              "Service not found",
              "The service you requested was not found in the system."
            )
          ),
          TE.map(v => ResponseSuccessJson(retrievedServiceToApiService(v)))
        )
      ),
      TE.toUnion
    )();
```

Reply:
It's very old code I didn't want to change in this iteration.

Reply:
I understand. Just fyi I tried the above refactoring and it passed tests. But I strongly agree with your opinion.

Reply:
Make a commit on this branch!

Reply:
done in https://github.com/pagopa/io-functions-admin/pull/165/commits/2c8dd170d2f3d6ecb3e84762da7e6f59eb45b51c